app/types/notificationType.js

```diff
@@ -1,6 +1,9 @@
 // @flow
+import type { ReactIntlMessage } from './i18nTypes';
+
 export type Notification = {
   id: string,
+  message: ReactIntlMessage,
   duration: ?number,
-  secondsTimerInterval: ?IntervalID,
+  secondsTimerInterval?: ?IntervalID,
```

Comment:
Is there any way to avoid having the property as optional and so their value? I think it’s kind of messy

Reply:
Yes It's messy.
As I describe in the first top comment, we need total refactoring for Notification API.
It's whole messy.

I tried my best what I could correct in given time and situation.